pkg/stub/configmap.go

```diff
@@ -166,35 +166,35 @@ func (h *Handler) storeImageStreamTagError(isName, tagName, message string) erro
 	return err
 }
 
-func (h *Handler) clearImageStreamTagError(isName, tagName string) error {
+func (h *Handler) clearImageStreamTagError(isName string, tags []string) error {
 	cm, err := h.configmapclientwrapper.Get(isName)
 	if err != nil {
 		if !kerrors.IsNotFound(err) {
 			logrus.Warningf("unexpected error on get of configmap %s: %s", isName, err.Error())
 			return err
 		}
-		cm := &corev1.ConfigMap{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      isName,
-				Namespace: v1.OperatorNamespace,
-			},
-			Data: map[string]string{},
-		}
-		cm, err = h.configmapclientwrapper.Create(cm)
-		if err != nil && !kerrors.IsAlreadyExists(err) {
-			return err
-		}
+		logrus.Printf("clearImageStreamTagError: stream %s already deleted so no worries on clearing tags",
+			isName)
```

Comment:
nit: This shouldn't need to be on it's own line.

Reply:
part of next push